### PR TITLE
Suggest

### PR DESCRIPTION
**Pull request by sokrahta**
Wabisabi should totally implement suggest. Elasticsearch has a fancy _suggest endpoint that leverages lucene index to give completion suggestions of various kinds depending on _mapping and analyzers. Details in the [documentation](http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/search-suggesters-completion.html).

[Review on Reviewable](https://reviewable.io/reviews/gphat/wabisabi/18)

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -22,4 +22,4 @@ libraryDependencies += "com.netaporter" %% "scala-uri" % "0.4.3"
 
 publishTo := Some(Resolver.file("file",  new File( "/Users/gphat/src/mvn-repo/releases" )) )
 
-
+resolvers += "Scalaz Bintray Repo" at "http://dl.bintray.com/scalaz/releases"
```
Comment (gphat) [inline build.sbt]: Why was this added?

**Comment by gphat** [comment]
This looks fine, but I'd like to know why you needed to add the bintray resolver before I merge in case there's something amiss.

**Comment by gphat** [comment]
I saw the scalar dependency failing when traffics this morning, so I'll assume that's why this came up!

**Comment by sokrahta** [comment]
there was an unresolved dependency to ```scalaz-stream``` - it must be transitive from another lib
